src/views/app/MainView.tsx

```diff
@@ -1,9 +1,17 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { addUser } from '../../redux/actions/userActions';
 import { Text, Button, Platform, SafeAreaView } from 'react-native';
 import { useNavigation } from '../../hooks/navigation-hooks';
 import { HandleAndroidBackButton } from '../../hooks/handle-android-back-button-hook';
 
-const MainView: React.FC = () => {
+export interface StoreProps {
```

Comment:
This is an opinionated comment. I tend to use interfaces where I'm doing OOP for example with classes. When I'm doing functional I just use types. There is not much of a difference with using type vs interface here (besides semantics).

Reply:
Also don't know if this needs to be exported.